sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerClientJavaDocCodeSnippets.java

```diff
@@ -119,7 +119,7 @@ public void beginRecognizeCustomFormsWithOptions() throws IOException {
 
         formRecognizerClient.beginRecognizeCustomForms(new RecognizeCustomFormsOptions(targetStream,
             form.length(), modelId).setFormContentType(FormContentType.IMAGE_JPEG)
-            .setIncludeTextContent(includeTextContent).setPollInterval(Duration.ofSeconds(5))).getFinalResult()
+            .setIncludeFieldElements(includeTextContent).setPollInterval(Duration.ofSeconds(5))).getFinalResult()
```

Comment:
nit: might need to consider local var `includeTextContent` to 'includeFieldElements' for the renaming as well.

Reply:
updated!!